Lib/fontTools/ttLib/tables/_n_a_m_e.py

```diff
@@ -607,34 +607,40 @@ def fromXML(self, name, attrs, content, ttFont):
 
     def __lt__(self, other):
         if type(self) != type(other):
-            return NotImplemented
+            return NotImplemented()
```

Comment:
NotImplemented is a singleton value, not an exception class to be instantiated (not to be confused with NotImplementedError) so remove the `()`

Reply:
Yikes, think autocomplete done a goofy one on me here.

Reply:
Hm, yes. I remember having to read up on returning this instead of raising a NotImplementedError, confused the two syntaxes/types.